### ISSUE TITLE
Missing react-a11y converter: react-a11y-no-onchange

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typescript-eslint/tslint-to-eslint-config#901]
### 💥 Missing Converter

-   `tslint-to-eslint-config` version: 2.0.1
-   `typescript-eslint` version: 4.9.1

#### TSLint Rule Name

`react-a11y-no-onchange`

Per https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md, the equivalent rule is now `jsx-a11y/no-onchange`.

#### Package Source

https://github.com/teamairship/tslint-react-a11y

**Comment by JoshuaKGoldberg** [comment]
Per https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md this should be marked as a deprecated/obsolete rule.

#1063

**Comment by JoshuaKGoldberg** [comment]
Since there is no known rule equivalent, the converter for this rule should return `{}`.
